From 95f313be32200ce7601b16255ac595f7c8350a7b Mon Sep 17 00:00:00 2001
From: Erfan Arvan <ea442@njit.edu>
Date: Thu, 2 Jul 2026 20:29:22 +0000
Subject: [PATCH] Fix filename case in run_all.py

---
 run_all.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/run_all.py b/run_all.py
index 292b7098..e8a8b660 100644
--- a/run_all.py
+++ b/run_all.py
@@ -10,7 +10,7 @@ COMMANDS = [
     ["python", "plot_all.py"],
     ["python", "show_all_figs_data.py"],
     ["python", "analyze_student_factors.py"],
-    ["python", "compare_university1_university2_correlations.py"],
+    ["python", "compare_University1_University2_correlations.py"],
 ]
 
 
-- 
2.43.0

